packages/composites/anchor/src/endpoint/price/beth.ts

```diff
@@ -43,24 +43,32 @@ const getStETHExchangeRate = async (
     curvePoolAbi,
     provider,
   )
-  const result = await stEthPoolContract.get_dy(1, 0, ethers.BigNumber.from(10).pow(18))
+  const baseTokenIndex = 1 // stETH has a token index of 1 in the curve pool
+  const quoteTokenIndex = 0 // ETH has a token index of 0 in the curve pool
+
+  // Gets how much ETH the pool will return if 1 stETH is traded.  The pool uses 18dp so 1 = 10**18.
+  const result = await stEthPoolContract.get_dy(
+    baseTokenIndex,
+    quoteTokenIndex,
+    ethers.BigNumber.from(10).pow(18),
+  )
   throwErrorForInvalidResult(
     jobRunID,
     result,
-    `stETH/ETH Exchange Rate from Curve Pool address ${config.stEthPoolContractAddress}`,
+    `ETH/stETH Exchange Rate from Curve Pool address ${config.stEthPoolContractAddress}`,
```

Comment:
This comment was correct. Base = stETH; Quote = ETH.

Reply:
reverted the comment